nupic/algorithms/anomaly.py

```diff
@@ -140,6 +148,17 @@ def compute(self, activeColumns, predictedColumns,
     if self._movingAverage is not None:
       score = self._movingAverage.next(score)
 
+    # apply binary discretization if required
+    if self._binaryThreshold is not None:
```

Comment:
We always apply a threshold after the likelihood calculation. Thresholding the raw anomaly score (whether averaged or not) usually doesn't work.